bigframes/core/compile/scalar_op_compiler.py

```diff
@@ -1013,6 +1013,17 @@ def geo_area_op_impl(x: ibis_types.Value):
     return typing.cast(ibis_types.GeoSpatialValue, x).area()
 
 
+@scalar_op_compiler.register_unary_op(ops.geo_st_astext_op)
+def geo_st_astext_op_impl(x: ibis_types.Value):
+    return typing.cast(ibis_types.GeoSpatialValue, x).as_text()
+
+
+@scalar_op_compiler.register_unary_op(ops.geo_st_geogfromtext_op)
+def geo_st_geogfromtext_op_impl(x: ibis_types.Value):
+    # Casting x to a str by passing it to the new function, st_geogfromtext()
```

Comment:
I don't think this comment is fully accurate. We're casting from a str to geo. Also, in general comments should explain why we're doing something not what we're doing.

```suggestion
    # Ibis doesn't seem to provide a dedicated method to cast from string to geography,
    # so we use a BigQuery scalar function, st_geogfromtext(), directly.
```